prov/verbs/src/fi_verbs.c

```diff
@@ -94,6 +94,20 @@ struct util_prov fi_ibv_util_prov = {
 	.flags = 0,
 };
 
+void fi_ibv_fastlock_acquire(fastlock_t *lock)
+{
+	fastlock_acquire(lock);
+}
+
+void fi_ibv_fastlock_release(fastlock_t *lock)
+{
+	fastlock_release(lock);
+}
+
+void fi_ibv_fastlock_empty(fastlock_t *lock)
+{
+}
```

Comment:
Can we define these as common functions in ofi_lock.h?  I would also change the name from 'empty' to 'noop' and have separate definitions for acquire_noop/release_noop, to make it a little easier to use.

Reply:
Sure! This is great idea.